src/glb/glb.ts

```diff
@@ -0,0 +1,5 @@
+export class Glb {
+  static makeEnum<T extends { [index: string]: U }, U extends string>(x: T) {
+    return x
```

Comment:
I don't get it, what do you need it for?

Reply:
I need it for create literal object enum which has many advantages over the usual TS enum.
but first of all I wanted to ged rid of non-typed hardcoded strings in components.

this link provides more info about ts enums and its cons
https://maxheiber.medium.com/alternatives-to-typescript-enums-50e4c16600b1

Reply:
Nicely done! Didn't know about this technique.